configs/hlcm_owner.yaml

```diff
@@ -10,8 +10,9 @@ choosers_predict_filters: (tenure == 'own')
 
 alts_fit_filters: null
 
+alts_predict_filters: (tenure == 'own')
 #alts_predict_filters: (tenure == 'own' & deed_restricted == False)
-alts_predict_filters: null
+#alts_predict_filters: null
```

Comment:
Good point. It looks like the low-income/deed-restricted HLCM steps are not running in any of the current scenarios (I don't see them in `baus.py`), so I'm reluctant to re-enable those filters or model steps in this PR, just in case it leads to additional things we need to resolve.

I'll plan to do it in a separate PR. I think the steps will be:
- add `hlcm_owner_lowincome` and `hlcm_renter_lowincome` steps back in `baus.py`
- add deed-restriction filters in the four HLCM config files
- include tenure filters in the two low-income configs
- check all the empirics to make sure the deed restriction splits and tenure splits in the base data line up with each other in a reasonable way, and that things continue to line up as the model runs